### ISSUE TITLE
One time donation [Feature]

### ISSUE THREAD

**Issue by borislavstoychev** [podkrepi-bg/frontend#660]
Implement the new donation design from Figma: https://www.figma.com/file/MmvFKzUv6yE5U2wrOpWtwS/Podkrepi.bg?node-id=10372%3A31085
## One Time Donation
- [x] UI - all 4 steps done
- [x] First Step done
- [ ]  Second Step
     :x:  Login functionality
     :x: Google login button
     :x: New registration link
     :heavy_check_mark:  Anonymous fields 
- [ ] Third step
     :x:  Card payment 
     :heavy_check_mark: Bank payment
- [ ] Final step
     :x: On success step buttons and social icons not working
     :x: On fail step buttons not working
- [ ] Stepper information is not stored anywhere

**Comment by dimitur2204** [comment]
Regarding the second step. I think it should be skipped in general if you are logged in or it shouldn't even be shown even so that there are only 3.

Also is it even possible to Register or Log In a user without redirecting to keycloak @kachar and even if it is should that be the way we handle it on this step. Is that how it was decided with the UX team?

**Comment by kachar** [comment]
I had a discussion with the UX team to change the way the donation steps are shown to the user. You might find a version that I've created so the inputs that are logically linked are on the same page:

https://www.figma.com/file/MmvFKzUv6yE5U2wrOpWtwS/Podkrepi.bg?node-id=10988%3A35233

A the moment the functional login/registration are only in Keycloak

This topic has been addressed multiple times but as it seems we're not able to re-create the cookies that Keycloak sets on the frontend. 

Related to 
- #408
- #259 
- #371 
- #417 

It's going to be a super major upgrade of the app if we're able to achieve that. Right now we've got the backend endpoints ready at `/login` and `/register`:

```
curl --request POST \
  --url http://localhost:5010/api/register \
  --header 'Content-Type: application/json' \
  --data '{
	"email": "test@example.com",
	"password": "test123456",
	"firstName": "Test",
	"lastName": "Test"
}'
```

```
curl --request POST \
  --url http://localhost:5010/api/login \
  --header 'Content-Type: application/json' \
  --data '{
	"email": "test@example.com",
	"password": "test123456",
}'
```
